packages/vega-functions/src/functions/format.js

```diff
@@ -1,6 +1,6 @@
 const wrap = method => function(value, spec) {
   const locale = this.context.dataflow.locale();
-  return locale[method](spec)(value);
+  return value === null ? 'null' : locale[method](spec)(value);
```

Comment:
This seems a little bit hacky, but is also the most minimal amount of code to make this work.